Dockerfile

```diff
@@ -13,4 +21,4 @@ RUN pip install -r requirements.txt
 
 COPY . /code/
 
-CMD ["python", "example.py"]
+CMD ["python3", "examples/conversion.py"]
```

Comment:
I think we shouldn't change CMD command to "examples/conversion.py", because the execution time could be too long as the Process takes to much resources. Just leave  "examples/example.py" , to show that loading is working.

Reply:
Done:
https://github.com/QuantuMobileSoftware/sentinel2tools/pull/7/commits/124629fc09eec81630cf7733ead8cb4333cbfb0d